packages/unitables-dmn/src/DmnRunnerOutputs.tsx

```diff
@@ -18,184 +18,47 @@ import * as React from "react";
 import { useMemo } from "react";
 import { DmnBuiltInDataType } from "@kie-tools/boxed-expression-component/dist/api";
 import { DmnUnitablesJsonSchemaBridge } from "./uniforms/DmnUnitablesJsonSchemaBridge";
-import { DMN_RUNNER_OUTPUT_COLUMN_MIN_WIDTH } from "./DmnRunnerOutputsTable";
 import { DecisionResult, DmnInputFieldProperties } from "@kie-tools/extended-services-api";
 
 interface OutputField {
+  type: string;
   dataType: DmnBuiltInDataType;
   width?: number;
   name: string;
   joinedName: string;
+  properties?: Record<string, any>;
+  items?: Record<string, any>[];
 }
```

Comment:
```suggestion
interface OutputFields {
  type: string;
  dataType: DmnBuiltInDataType;
  width?: number;
  name: string;
  joinedName: string;
  properties?: Record<string, any>;
  items?: Record<string, any>[];
}
```

Then delete the `export type OutputFields = OutputField;`

Reply:
@thiagoelg Thanks for your review! Actually, I forgot to remove the `OutputFields`. I've kept the `OutputField`, and removed all `OutputFields` references.